package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "tabalanche-extension",
+  "version": "1.0.0",
+  "description": "[![Join the chat at https://gitter.im/tabalanche/tabalanche-extension](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/tabalanche/tabalanche-extension)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "web-ext build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eight04/tabalanche-extension.git"
```

Comment:
`package.json` is generated by `npm init -y`. You may want to modify some metadata e.g. `version`, `repository`, `author`, etc.